hail/src/main/scala/is/hail/io/fs/TerraAzureStorageFS.scala

```diff
@@ -25,7 +27,14 @@ object TerraAzureStorageFS {
 class TerraAzureStorageFS extends AzureStorageFS() {
   import TerraAzureStorageFS.{log, TEN_MINUTES_IN_MS}
 
-  private[this] val httpClient = HttpClients.custom().build()
+  private[this] val timeout = 30
+  private[this] val requestConfig = RequestConfig.custom()
+    .setConnectTimeout(timeout * 1000)
+    .setSocketTimeout(timeout * 1000)
```

Comment:
I was seeing occasional socket timeouts from calling Terra/WSM, this seems to help.